protocol/communities/manager.go

```diff
@@ -87,6 +88,25 @@ func NewManager(identity *ecdsa.PublicKey, db *sql.DB, logger *zap.Logger, verif
 	return manager, nil
 }
 
+type archiveMDSlice []*archiveMetadata
+
+type archiveMetadata struct {
+	hash string
+	from uint64
+}
+
+func (md archiveMDSlice) Len() int {
+	return len(md)
+}
+
+func (md archiveMDSlice) Swap(i, j int) {
+	md[i], md[j] = md[j], md[i]
+}
+
+func (md archiveMDSlice) Less(i, j int) bool {
+	return md[i].from > md[j].from
+}
```

Comment:
This is later used when we receive the archive index to make it sortable. This is needed to ensure latest archives are always downloaded first (instead of random).